specification/migrate/resource-manager/Microsoft.OffAzure/stable/2020-01-01/migrate.json

```diff
@@ -0,0 +1,4943 @@
+      "post": {
+        "tags": [
+          "Sites"
+        ],
+        "summary": "Method to get site health summary.",
+        "operationId": "Sites_GetSiteHealthSummary",
+        "consumes": [
+          
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "subscriptionId",
+            "in": "path",
+            "description": "The ID of the target subscription.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "The name of the resource group. The name is case insensitive.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "siteName",
+            "in": "path",
+            "description": "Site name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "description": "The API version to use for this operation.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SiteHealthSummaryCollection"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Get VMware site health summary.": {
+            "$ref": "./examples/VMwareSiteHealthSummary_Post.json"
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OffAzure/VMwareSites/{siteName}/vcenters/{vcenterName}": {
+      "get": {
+        "tags": [
+          "VCenter"
+        ],
+        "summary": "Method to get a vCenter.",
+        "operationId": "VCenter_GetVCenter",
+        "consumes": [
+          
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "subscriptionId",
+            "in": "path",
+            "description": "The ID of the target subscription.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "The name of the resource group. The name is case insensitive.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "siteName",
+            "in": "path",
+            "description": "Site name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "vcenterName",
+            "i
```

Comment:
What does 'other' mean? Is there a better description of the type of database you discover? Could you use a name based on how you discovered the database?

Reply:
We do little deep discovery of sql server database. We only do high level discovery of other databases such as apache couch db, mongodb, postgres etc.
As we enhance this feature for deep discovery of databases, we will keep on removing that database server from other databases data model and have specific model for it, similar to sql server.

Reply:
Will take it during GA of AppsAndRoles after we have more clarity on it.